src/server/pegasus_service_app.h

```diff
@@ -47,6 +48,7 @@ class pegasus_replication_service_app : public ::dsn::replication::replication_s
 
         if (ret == ::dsn::ERR_OK) {
             pegasus_counter_reporter::instance().start();
```

Comment:
It's strange a singleton to explicit start, we can improve it later, define it as a common object too?

Reply:
Actually all codes about perf counters will be removed after all metrics are migrated. I'll add a TODO here to comment.